runtime/lib/backend/lightning/lightning_dynamic/LightningSimulator.cpp

```diff
@@ -133,6 +136,28 @@ void LightningSimulator::NamedOperation(const std::string &name, const std::vect
     if (this->tape_recording) {
         this->cache_manager.addOperation(name, params, dev_wires, inverse);
     }
+    //}
+    /*
+        else if (name == "ISWAP") {
+            RT_FAIL_IF(params.size() != 0, "Number of parameters for ISWAP gate should be zero");
+            std::vector<std::complex<double>> matrix{
+                {1, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 1}, {0, 0},
+                {0, 0}, {0, 1}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {1, 0},
+            };
+            LightningSimulator::MatrixOperation(matrix, wires, inverse);
+        }
+        else if (name == "PSWAP") {
+            RT_FAIL_IF(params.size() != 1, "Number of parameters for PSWAP gate should be one");
+            double phi = params[0];
+            const std::complex<double> i{0.0, 1.0};
+            std::complex<double> eiphi = std::exp(i * phi);
+            std::vector<std::complex<double>> matrix{
+                {1, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, eiphi,  {0, 0},
+                {0, 0}, eiphi,  {0, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {1, 0},
+            };
+            LightningSimulator::MatrixOperation(matrix, wires, inverse);
+        }
+    */
```

Comment:
You can remove these commented lines. This issue will be resolved upstream in the pennylane-lightning repository.

Reply:
done